beacon-chain/forkchoice/doubly-linked-tree/epbs.go

```diff
@@ -0,0 +1,9 @@
+package doublylinkedtree
+
+func (n *Node) isParentFull() bool {
+	// Finalized checkpoint is considered full
+	if n.parent == nil || n.parent.parent == nil {
```

Comment:
The finalized checkpoint itself is full and it's parent is considered full. Any child of finalized will have it's parent full as well